terraform/groups/ecs-service/data.tf

```diff
@@ -42,6 +42,15 @@ data "aws_lb_listener" "service_lb_listener" {
   port = 443
 }
 
+data "aws_lb" "secondary_lb" {
```

Comment:
There is already a reference to the alb (data.aws_lb.service_lb), so we can reuse that